assets/app/views/_overview-deployment.html

```diff
@@ -72,22 +74,27 @@ <h3 class="h4">{{rc.metadata.name}}</h3>
     </div>
   </div>
 
-  <div class="pod-block">
-    <div class="component-label">Pod Template</div>
+  <div class="pod-block row">
+    <div class="col-xs-12 col-lg-6">
+      <div class="component-label">Pod Template</div>
+      <pod-template
+        pod-template="rc.spec.template"
+        images-by-docker-reference="imagesByDockerReference"
+        builds="builds">
+      </pod-template>
+    </div>
 
-    <pod-template
-      pod-template="rc.spec.template"
-      images-by-docker-reference="imagesByDockerReference"
-      builds="builds">
-    </pod-template>
+    <div class="col-xs-12 col-lg-6">
+      <div class="component-label">
+        <a href="javascript:;" ng-click="viewPodsForDeployment(rc)">
```

Comment:
I thought that use to cause the link to be disabled in some browsers but
apparently not.

On Thu, Sep 3, 2015 at 11:04 AM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/views/_overview-deployment.html
> <https://github.com/openshift/origin/pull/4486#discussion_r38655885>:
>
> >
> > -    <pod-template
> > -      pod-template="rc.spec.template"
> > -      images-by-docker-reference="imagesByDockerReference"
> > -      builds="builds">
> > -    </pod-template>
> > +    <div class="col-xs-12 col-lg-6">
> > +      <div class="component-label">
> > +        <a href="javascript:;" ng-click="viewPodsForDeployment(rc)">
>
> Any reason to use href="javascript:;" vs just href=""?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4486/files#r38655885>.
>